Automattic/jetpack
# AI Breve: Update highlight logic to use RichText

Fixes problems related with Breve usage since it was not native integrated with Gutenberg.

## Proposed changes:

* Refactor `highlights` to use `registerFormatType`
* Add `Breve` store.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Activate `Breve`.
* Go to your editor.
* Add some content and make sure it includes some `Complex Words`.
* It should highlight these texts.
* Hovering it should show the Popover.
